src/test/java/pdfverification/tests/HgvPRSBilingualTests.java

```diff
@@ -0,0 +1,57 @@
+package pdfverification.tests;
+
+import com.github.jknack.handlebars.Handlebars;
+import com.itextpdf.text.pdf.PdfReader;
+import htmlverification.service.CvsCertificateTestDataProvider;
+import org.junit.Before;
+import org.junit.Test;
+import org.xhtmlrenderer.pdf.ITextRenderer;
+import pdfverification.service.PDFParser;
+import uk.gov.dvsa.model.cvs.CvsMotCertificate;
+import uk.gov.dvsa.service.HtmlGenerator;
+import uk.gov.dvsa.service.PDFGenerationService;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertTrue;
+
+public class HgvPRSBilingualTests {
+    private static final String CERT_NAME = "MOT test certificate (HGV)";
```

Comment:
IT's probably not worth making these variables, they're only being used in one place currently

Reply:
Yes that's fine i've no preference.